consumers/campaigner/main.go

```diff
@@ -44,20 +47,41 @@ func (h *MessageHandler) HandleMessage(m *nsq.Message) error {
 		return nil
 	}
 
-	// fetching subs that are active and that have not been blacklisted
+	campaign, err := h.s.GetCampaign(msg.CampaignID, msg.UserID)
+	if err != nil {
+		logrus.WithFields(logrus.Fields{
+			"campaign_id": msg.CampaignID,
+			"user_id":     msg.UserID},
+		).WithError(err).Error("unable to find campaign")
+		return nil
+	}
+	if campaign.Status != entities.StatusDraft {
+		logrus.WithError(err).
+			WithField("status", campaign.Status).
```

Comment:
Add more fields from `msg` here, like user_id, campaign_id etc. Also the wording on the message could be a bit better, for example: Errorf("potentially duplicate message: campaign status is '%s', it should be 'draft'", campaign.Status)